quark/utils/tools.py

```diff
@@ -55,3 +55,40 @@ def descriptor_to_androguard_format(descriptor):
     new_descriptor = re.sub(r"\[ ", "[", new_descriptor)
 
     return new_descriptor
+
+
+def api_filter(data, api_pool, percentile_rank=0.2):
```

Comment:
I suggest renaming this function to `filter_api_by_usage_count` to say more about what the function does.